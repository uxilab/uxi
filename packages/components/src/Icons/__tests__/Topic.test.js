import React from 'react';
import Topic from '../Topic';

describe('Icon : <Topic />', () => {
  it('match snapshot', () => {
    expect(shallow(<Topic />)).toMatchSnapshot();
  });
});

