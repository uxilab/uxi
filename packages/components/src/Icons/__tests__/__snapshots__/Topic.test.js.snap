// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon : <Topic /> match snapshot 1`] = `
<SvgIcon
  color="#6d6d71"
  hoverColor="#6d6d71"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
  size={24}
  style={Object {}}
  viewBox="0 0 24 24"
>
  <svg
    focusable="false"
    height="24px"
    viewBox="0 0 16.49 16"
    width="24px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Layer 2"
      id="Layer_2"
    >
      <g
        data-name="Layer 1"
        id="Layer_1-2"
      >
        <path
          d="M3.55,16a.37.37,0,0,0,.39,0l3.7-2.78h7.29a1.56,1.56,0,0,0,1.55-1.55v-10A1.56,1.56,0,0,0,14.94,0H1.55A1.56,1.56,0,0,0,0,1.55v10a1.56,1.56,0,0,0,1.55,1.55H3.35v2.48A.37.37,0,0,0,3.55,16ZM1.43,11.71V1.43H15.06V11.71H7.17L4.78,13.51V11.71Z"
        />
        <circle
          cx="6"
          cy="6.7"
          r="1.3"
        />
        <circle
          cx="10.66"
          cy="6.7"
          r="1.3"
        />
      </g>
    </g>
  </svg>
</SvgIcon>
`;
